### DISCUSSION TITLE
intraday steps

### DISCUSSION THREAD

**Discussion by DREJAG** [mrkaye97/fitbitr#7] [Ideas]
Hi, is it possible to add a function to extract intraday steps, similar to intraday HR, in time intervals of 1 min?

**Comment by mrkaye97** [comment]
this is a great idea! I'll include this in the next version I'm working on which will hopefully release in the next few weeks

**Comment by mrkaye97** [comment]
@DREJAG this is live in version 0.3.0, which I just released!

**Reply by DREJAG** [reply]
Great. Thanks!
